config-user.default.php

```diff
@@ -62,6 +62,7 @@
 		'site' => true,
 	),
 	'theme' => 'Origine',
+	'darkMode' => false,
```

Comment:
Do we maybe want three states: { yes, no, automatic } ?

Reply:
Agreed, probably with automatic as the default?

Reply:
Maybe later.
`yes` would mean to find another solution than the media query `@media (prefers-color-scheme: dark)`.

Reply:
There is a simple approach on https://stackoverflow.com/questions/56300132/how-to-override-css-prefers-color-scheme-setting/68824350#68824350
But it requires having two copies of the variables, which is not super elegant.

Reply:
I hope we can find a satisfying approach for { auto, yes, no, }, but if we do not implement *yes* for now, I think in any case we should have { auto, no, }, default being auto.

Reply:
Definitely not a hold-up from my end; I was just agreeing it'd be nice to have. The SO link there looks reasonably elegant; I can't think of anything better.

Reply:
yes, this is the only solution, as far as I know.
With SCSS we could keep the maintenance effort small.

Reply:
`no` and `auto` is now implemented